packaging/linux/deb/package_binaries.sh

```diff
@@ -81,6 +81,14 @@ build_one_architecture() {
     > "$cron_file"
   chmod 755 "$cron_file"
 
+  # Whitelist for NativeMessaging
+  kbnm_bin="/usr/bin/kbnm"
+  kbnm_file="$dest/etc/opt/chrome/native-messaging-hosts/io.keybase.kbnm"
+  mkdir -p "$(dirname "$kbnm_file")"
+  cat "$here/host_json.template" \
+    | sed "s|@@HOST_PATH@@|/usr/local/bin/$kbnm_bin|g"
+  chmod 644 "$kbnm_file"
```

Comment:
I figure it's more consistent with the other blocks above, and explicit is better than implicit, but I agree it's superfluous so removed for now.